gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Dec 14 22:58:07 CET 2015
+#Fri Nov 11 11:39:09 PST 2016
```

Comment:
Note - it "didn't quite take" because the task wasn't included in the build.gradle file.